src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -58,6 +60,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@mkordas lets go with a puzzle here, this PR already adds one new class. If it now also starts refactoring this class it's just too much :)